.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+
+<!--
+Thank you for your interest in contributing to XCSoar! Please fill out the
+following questions to make it easier for us to review your changes.
+
+You do not need to check all the boxes below all at once, feel free to take
+your time and add more commits. If you're done and ready for review, please
+check the last box.
+-->
+
+
+What is the purpose of this pull request? What does it change?
+--------------------------------------------------------------
+
+<!--
+Please describe the changes here, as detailed as needed.
+-->
+
+
+Has the changes been discussed in an issue or the forum?
+--------------------------------------------------------
+
+<!--
+Please link issues and relevant forum posts here.
+
+If this PR resolves an issue on GitHub, write "Closes #1234" so that the issue
+is closed automatically when this PR is merged.
+-->
+
+
+Checklist
+---------
+
+- [ ] I have enabled [maintainer edits for this PR](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)
+- [ ] I have added tests for all changes in this PR
```

Comment:
We have unit tests only for a tiny fraction of the source code - how are we expecting people to add unit tests for UI changes?

Reply:
I wasn't sure what tests there are that we can request from people. I guess this should be removed then :)

Reply:
It **is** good to have unit tests - and if possible, there should be unit tests for new code - but most of the time, it isn't possible. We can test device drivers, and we can test math formulas, but we can't test the bulk of the code.

Reply:
Removed.